README.md

```diff
@@ -50,7 +52,9 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+residents.pop([1])
+#something
+guests.push("Belle")
 ```
 
```

Comment:
you should test these out to see if you're accessing what you think you are. For example, you can't get to residents without referencing town. here's a sample solution: 
`belle = town[:residents][1]`
`town[:castle][:guests] = belle`
`town.delete([:residents][1])`